spec/system/task_spec.rb

```diff
@@ -2,23 +2,23 @@
 
 RSpec.describe 'Task', type: :system do
   describe 'Task一覧' do
+      let(:project) { create(:project) }
+      let!(:task) { create(:task, project_id: project.id) }
```

Comment:
`association`の定義をつけると`project_id`の指定が不要になりますね。
```suggestion
      let!(:task) { create(:task) }
```

Reply:
task.rbにprojectモデルのassociationの定義をしました。352995a95294b8abf7221b2c730914bc64e436a7
associationをしたのでproject_idの指定を消去しました。cb15f74d856e88acd29e6ae7637d87a912905ffb

Reply:
352995a95294b8abf7221b2c730914bc64e436a7